src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
make a global Random field and use it here

Reply:
[Random field declared](https://github.com/Maksym-Kulakov/jv-oop-advanced/commit/d9bf0e26cd1a600492f3ae205110cdef86fb0772)